Server/src/Models/Business.js

```diff
@@ -0,0 +1,8 @@
+const mongoose = require('mongoose');
+
+// Schema
+const business_schema = mongoose.Schema({
+    name: {type: String, required: true}
```

Comment:
Parent schema should hold more global/general info that applies to all business entities